core/server/api/v2/utils/serializers/output/all.js

```diff
@@ -0,0 +1,25 @@
+const debug = require('ghost-ignition').debug('api:v2:utils:serializers:output:all');
+const _ = require('lodash');
+
+const removeXBY = (object) => {
+    _.each(object, (value, key) => {
+        // CASE: go deeper
+        if (_.isObject(object[key]) || _.isArray(object[key])) {
+            removeXBY(object[key]);
+        } else if (['updated_by', 'created_by', 'published_by'].includes(key)) {
+            delete object[key];
+        }
+    });
+
+    return object;
+};
+
+module.exports = {
+    after(apiConfig, frame) {
+        debug('all after');
+
+        if (frame.response) {
+            frame.response = removeXBY(frame.response);
```

Comment:
what a shitty variable name 🤣

Reply:
Was thinking about it as well... in general, this method is not just about 'x_by' properties. To me, it looks more like a 'property filter'. Can imagine a situation where these fields would be expanded with e.g.: 'updated_at'